add: two new RDP scriptland events for negotiation flags

anthonykasza opened the pull request:
This is a pretty simple change. 2 new events are raised by the RDP analyzer.

`rdp_connect_request_flags` is raised in tandem with the `rdp_connect_request` event and `rdp_negotiation_response_flags` is raised in tandem with `rdp_negotiation_response`.

Ideally, `rdp_connect_request` and `rdp_negotiation_response` parameters would be extended to include the flags included in each of those PDUs but I believe that may break things.

0xxon commented:
I think with https://github.com/zeek/zeek/pull/892 merged, we can now  extend rdp_connect_request/rdp_negotiation_response without breaking everything. And that looks preferable to me.

anthonykasza commented:
Thanks @0xxon.
I've updated my changes to make use of alternate event prototypes. I also noticed a third event, `rdp_negotiation_failure`, which could include flags and so I added them.
